src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -208,7 +208,7 @@ public function has(string $id): bool
     public function get(string $id, int $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE): ?object
     {
         return $this->services[$id]
-            ?? $this->services[$id = $this->aliases[$id] ?? $id]
```

Comment:
this was not dead code, as this could change the value of `$id` used on the next line

Reply:
You're right !